EIPS/eip-ecmul-precompile.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: To be assigned
+title: Precompile for Elliptic Curve Linear Combinations
+discussions-to: https://ethereum-magicians.org/t/ewasm-precompile-for-general-elliptic-curve-math/2581
+status: Draft
+type: Core
+replaces: EIP-196 (in some cases), EIP 665, EIP 1108 (partially), https://github.com/ethereum/EIPs/issues/603
```

Comment:
Unfortunately this can only use an EIP number and that must be merged (cannot be an unmerged draft). For #603 I'd suggest you refer to it in a "References" or "Prior art" section.

Reply:
I actually don't think the `replaces` field is appropriate here. Perhaps it is as field not applicable to the EIP process we're following.